Namco/pacman.py

```diff
@@ -1,76 +1,102 @@
 import numpy as np
-from pytimedinput import timedInput
+import os
+
+# Constants
+PACMAN = 1
+OBJECT = 2
+
 
 class PacMan:
-    def __init__(self, map_size=10, max_time_step=100):
+    def __init__(self, map_size=10, max_time_step=100, num_of_objects=10):
```

Comment:
Az input értékeket (10,100,10) a main függvényben az instantiatingnél is megadjuk.
env = PacMan(map_size=10,
                 max_time_step=100,
                 num_of_objects=10)
Ott kéne csak nem?

Reply:
Az itt megadott értékek default-ak. Ezt fogja használni a program, ha lent nem adunk meg semmit. De ha lent megadunk valamit, akkor az fog érvényesülni.